pkg/executor/bacalhau/bacalhau.go

```diff
@@ -26,221 +39,238 @@ type BacalhauExecutorOptions struct {
 }
 
 type BacalhauExecutor struct {
-	Options        BacalhauExecutorOptions
-	bacalhauEnv    []string
+	Options     BacalhauExecutorOptions
+	bacalhauEnv []string
```

Comment:
Please use `go fmt` so we don't have to sort through stuff like this from your editor 🙏